src/App.js

```diff
@@ -1,9 +1,89 @@
 import React from 'react';
 import './App.css';
+import Buttons from "./Buttons";
+import Products from "./Products";
+
+import bag_1 from "./assets/bag_1.png";
+import bag_2 from "./assets/bag_2.png";
+import bag_3 from "./assets/bag_3.png";
+import bag_4 from "./assets/bag_4.png";
+import our_story from "./assets/our_story.png";
+import brand from "./assets/brand.png";
+
+import Tiles from "./Tiles";
+
 
 function App() {
   return (
-      <h1>Handbags & Purses</h1>
+<>
+    <header>
+        <h1>Handbags & Purses</h1>
+           <nav>
+            <Buttons
+               title="to the collection"
+            />
+            <Buttons
+                title="shop all bags"
+            />
+            <Buttons
+                title="pre-orders"
+                disabled="disabled"
+            />
+          </nav>
+    </header>
+
+      <main>
+        <Products
+            imageurl={bag_1}
+            label="Best seller"
+            title="The handy bag"
+            price="€400,-"
```

Comment:
Het is ook mogelijk om in het component alleen een getal of cijfer mee te geven. Dan zou je de overige '€' en '.-' op deze manier toevoegen: <h4'> €{price},- </h4'>